homework3/semi-uber/Server/api/handlers/loginHandlers/loginPostHandler.js

```diff
@@ -25,18 +26,16 @@ const loginPostHandler = async (req, res) => {
   }
 
   try {
-    await bcrypt.compare(password, user.password);
+    comparedPass = await bcrypt.compare(password, user.password);
```

Comment:
please check if we need try/catch for the `bcrypt.compare`, maybe it is redundant here

Reply:
As i see, there is possible to use a callback function with error handling as 3rd argument of function, so i guess, there is can be error 
![image](https://user-images.githubusercontent.com/54358514/78457003-e0238500-76af-11ea-81a9-3c9a1e93b7ed.png)